integration/mediation-tests/tests-mediator-2/src/test/resources/artifacts/ESB/server/repository/deployment/server/synapse-configs/default/proxy-services/scriptMediatorNashornJsInlineTestProxy.xml

```diff
@@ -0,0 +1,36 @@
+<!--
+  ~ Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+  ~
+  ~ WSO2 Inc. licenses this file to you under the Apache License,
+  ~ Version 2.0 (the "License"); you may not use this file except
+  ~ in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied. See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+
+<proxy xmlns="http://ws.apache.org/ns/synapse" name="scriptMediatorNashornJsInlineTestProxy" transports="http">
+    <target>
+        <inSequence>
+            <!-- transform the custom quote request into a standard quote request expected by the service -->
+            <script language="nashornJs" key="stockQuoteNashornJsScript" function="transformRequest"/>
+            <send>
```

Comment:
Do we really need to forward the request to an endpoint? Can't we respond right after the script mediation and evaluate the response?